groups/groups.yaml

```diff
@@ -138,36 +119,6 @@ groups:
       - jim@nirmata.com
       - rficcaglia@gmail.com
 
-  - email-id: election@kubernetes.io
-    name: election
-    description: |-
-      Kubernetes elections committee
-    settings:
-      WhoCanPostMessage: "ANYONE_CAN_POST"
-      MembersCanPostAsTheGroup: "true"
-      ReconcileMembers: "true"
-    owners:
-      - ihor@cncf.io
-      - josh@berkus.org
-      - jdumars@gmail.com
-
-  - email-id: github@kubernetes.io
-    name: github
-    description: |-
-      Kubernetes GitHub Admins
-    settings:
-      WhoCanPostMessage: "ANYONE_CAN_POST"
-      ReconcileMembers: "true"
-    owners:
-      - cblecker@gmail.com
-      - spiffxp@gmail.com
-      - spiffxp@google.com
-    managers:
-      - fejta@google.com
-      - ihor.dvoretskyi@gmail.com
-      - nikitaraghunath@gmail.com
-      - killen.bob@gmail.com
-
   # Every RBAC group should be added here.
   - email-id: gke-security-groups@kubernetes.io
```

Comment:
@spiffxp why not move it to `groups/wg-k8s-infra` ?

Reply:
wasn't sure if it was simpler to leave at root or move in k8s-infra so punted for this first PR